aws/resource_aws_cognito_identity_pool.go

```diff
@@ -218,9 +225,19 @@ func resourceAwsCognitoIdentityPoolUpdate(d *schema.ResourceData, meta interface
 		params.SamlProviderARNs = expandStringList(v.([]interface{}))
 	}
 
+	if v, ok := d.GetOk("tags"); ok {
+		params.IdentityPoolTags = tagsFromMapGeneric(v.(map[string]interface{}))
+	}
+
+	log.Printf("[DEBUG] Updating Cognito Identity Pool: %s", params)
+
+	if err := setTagsCognito(conn, d); err != nil {
```

Comment:
Nit: Can you please move this below the `UpdateIdentityPool` API call to not interrupt the flow of the function? Thanks.

Reply:
fixed in 4ed6457ff